apiary.apib

```diff
@@ -3315,6 +3317,151 @@ Get information about a meeting.
                     + open
                     + done
 
+## Calls [/calls]
+
+### calls.add [POST /calls.add]
+
+Add a new call.
+
++ Request (application/json)
+
+    + Attributes
+        + description: `Description of the call` (string, optional)
+        + customer (object, required)
+            + type (enum, required)
+                + Members
+                    + company
+                    + contact
+            + id: `2659dc4d-444b-4ced-b51c-b87591f604d7` (string, required)
+        + due_date: `2016-02-04T16:00:00+00:00` (string, required)
+        + responsible_user_id: `98b2863e-7b01-4232-82f5-ede1f0b9db22` (string, required)
+        + custom_fields (array[CustomFieldValue], optional)
+
++ Response 201 (application/json)
+
+    + Attributes (object)
+        + data (object)
+            + type: `call` (string)
+            + id: `65a35860-dcca-4850-9fd6-47ff08469e0c` (string)
+
+### calls.list [POST /calls.list]
+
+Get a list of calls.
+
++ Request (application/json)
+    + Attributes (object)
+        + filter (object, optional)
+            + start_date: `2021-11-21` (string) - Filter on dates that are on or after the given day
+            + end_date: `2022-05-13` (string)  - Filter on dates that are on or before the given day
+            + relates_to (object, optional) - Filter calls by related object (currently only companies allowed)
+                + id: `415fa7e0-e319-0952-b973-f067f2b49d56` (string)
+                + type: `company` (string)
+            + milestone_id: `f29abf48-337d-44b4-aad4-585f5277a456` (string, optional) - Filter calls by milestone
```

Comment:
Q: Let's drop milestone since this is no longer a thing in next-gen?
Unless it is a requirement then 👍

Reply:
Already removed in the follow-up.
https://github.com/teamleadercrm/api/pull/749/commits/d8d1ede67cf97d5ca8bf727d581ab165daf0b620